Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1516,8 +1520,6 @@ internal extension Purchases {
         return self.systemInfo.storeKit2Setting
     }
 
-    #if DEBUG
-
```

Comment:
`fetchReceipt` is already in an `internal` extension.

Reply:
This was only used for tests, and I think we didn't want to use it in production until we verified if it was too slow.
Let's at least make sure we don't call this in the main thread with `RCTestAssertNotMainThread`